index.html

```diff
@@ -218,9 +231,9 @@ <h1 class="heading1"  >Contact me</h1>
     <input type="text" value="Diana" name="name" maxlength="30"  placeholder="Name" class="input text" required/>
     <input type="email" value="dianab21@gmail.com" name="email" placeholder="Email" class="input text" required/>
     <textarea id="message" cols="30"  maxlength="500"   rows="9" placeholder="Write Your Message Here" required></textarea> 
-
+​
     <button type="submit" class="form-btn">Get In Touch</a></button>
   </form>
```

Comment:
- [ ] Kindly fix the background image on the contact section to match the given template. I see you added the background image for the mobile on the desktop too. Consider downloading the desktop background and add it to your work.
### Your design
![image](https://user-images.githubusercontent.com/94551484/221290930-1ad9bf33-054b-41e8-946f-559b8cff4cd9.png)
### Figma template
![image](https://user-images.githubusercontent.com/94551484/221291004-43257a6a-387d-4da1-9121-dcca26c3832a.png)